cds/modules/deposit/static/js/cds_deposit/avc/avc.module.js

```diff
@@ -216,19 +216,10 @@ angular.module('schemaForm')
 
       // Use this only in multiple select
       if ($scope.form.type === 'uiselectmultiple') {
-        $scope.$watchCollection('ngModel.$modelValue', function(newValue) {
-          if(newValue !== undefined && !_.isEqual($scope.form.$$selectedObjects, newValue)){
-            $scope.form.$$selectedObjects = newValue;
-          }
-        }, true);
-
-        $scope.$watchCollection('form.$$selectedObjects', function(newValue) {
-          if(newValue !== undefined && !_.isEqual($scope.ngModel.$modelValue, newValue)) {
-            $scope.ngModel.$setViewValue(_.uniq(newValue));
-            $scope.ngModel.$commitViewValue();
-            $scope.ngModel.$render();
-          }
-        }, true);
+        var form = $scope.form;
```

Comment:
Always nice to remove 2 watchers! :)

I would have added a comment to explain why we need this...